bin/ta_puppet_report_viewer/modalert_generate_detailed_report_helper.py

```diff
@@ -142,22 +150,20 @@ def process_event(helper, *args, **kwargs):
     splunk_server = helper.get_global_setting("splunk_server")
     hec_token = helper.get_global_setting("splunk_hec_token")
 
-
-    pe_console = helper.get_global_setting("pe_console") or puppet_db_server
-
     resource_response = get_resource_events(uuid, puppet_db_server, auth_token)
 
     detailed_report = resource_response[0]
-    
-    certname = detailed_report['certname']
-    
-    facts= get_certname_facts(certname, puppet_db_server, auth_token)
 
-    detailed_report['facts'] = facts['facts']
+    mdict = collections.defaultdict(dict)
+
+    for m in detailed_report['metrics']['data']:
+      mdict[m['category']][m['name']] = m['value']
+    
+    detailed_report['metrics'] = dict(mdict)
 
     detailed_report['url'] = 'https://{}/#/inspect/report/{}/events'.format(pe_console, detailed_report["hash"])
 
-    detailed_report["puppetdb_callback_hostname"] = puppetdb_callback_hostname
+    detailed_report["pe_console"] = pe_console
```

Comment:
Nit alert: In this section, for style purposes, the use of single vs double quotes should be consistent.